project-base/ecs.php

```diff
@@ -143,80 +105,13 @@
+
             // @deprecated File is excluded as the comments are already missing and deprecated methods will not be in next major
             DeprecatedAnnotationDeclarationSniff::class => [
                 __DIR__ . '/tests/App/Test/Codeception/Module/StrictWebDriver.php',
             ],
-            // @deprecated This will be moved from project-base to coding-standards package in next major version
-            // rule breaks jms/translation-budle as it fails on this usage: `[, $b] = $var`
-            ListSyntaxFixer::class => null,
         ]
     );
 
-    $services->set(DeclareStrictTypesFixer::class);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(DisallowEqualOperatorsSniff::class);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(ValidClassNameSniff::class);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(NoUselessElseFixer::class);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(AssignmentInConditionSniff::class);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(DisallowEmptySniff::class);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(EarlyExitSniff::class)
-        ->property('ignoreStandaloneIfInScope', true)
-        ->property('ignoreOneLineTrailingIf', true)
-        ->property('ignoreTrailingIfWithOneInstruction', true);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(ParentCallSpacingSniff::class)
-        ->property('linesCountBeforeParentCall', 1)
-        ->property('linesCountAfterParentCall', 1);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(ReferenceUsedNamesOnlySniff::class)
-        ->property('allowPartialUses', true);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(DeprecatedAnnotationDeclarationSniff::class);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(DocCommentSpacingSniff::class)
-        ->property('linesCountBetweenDifferentAnnotationsTypes', 0);
-
-    // @deprecated This will be moved from project-base to coding-standards package in next major version
-    $services->set(UselessIfConditionWithReturnSniff::class);
-
-    $services->set(PhpdocToPropertyTypeFixer::class);
-
     $containerConfigurator->import(__DIR__ . '/vendor/shopsys/coding-standards/ecs.php', null, true);
```

Comment:
Maybe this import should be at the start of the file, because it includes sets which could override rules? Same for the project-base and other packages.

Reply:
What do qou think? @grossmannmartin